third_party/generate-xnnpack-wrappers.py

```diff
@@ -92,6 +92,11 @@
     # add non-prod microkernel sources here:
 }
 
+# Source files not needed in buck build.
+IGNORED_SOURCES = set([
```

Comment:
super nit: constant sequences should be tuples, not lists:
```
set((...,))
```
(note the trailing comma)

This is because tuples are actually compile-time constants whereas lists (being mutable) emit a tuple which is then used to build a transient list.